templates/terraform/main.tf

```diff
@@ -0,0 +1,81 @@
+provider "aws" {
+  region              = "${var.region}"
+}
+
```

Comment:
Let me know where we want to move these configs into later. I'm just putting everything into the top level `main.tf` for now